packages/presets/swc-plugin/package.json

```diff
@@ -27,32 +27,6 @@
     "swc_plugin.wasm"
   ],
   "type": "module",
```

Comment:
`type: module` wasn't in the original package.json.
I know this'd mark this package as ESM, but I'm not too familiar with how it's supposed to work with WASM? 😵 

It seems to work with or without this value, so I'm leaving it here, please let me know if we should remove this instead.